src/components/calculator.js

```diff
@@ -0,0 +1,37 @@
+import { Component } from 'react';
+import './calculator.css';
+/*eslint-disable */ //react/prefer-stateless-function
```

Comment:
- [OPTIONAL] Kindly do the same here 👍🏻

Reply:
Hi @BenMukebo I can make the change to use PureComponents but is still asks me for writing propTypes checks and destructuring, which is not required in this project.

Reply:
Hello @PinkMoon25, what kind of error are you getting? if it's about `propTypes` kindly fix this !!

Reply:
How do i check for propTypes if i have different value for prop.values

Reply:
- [ ] If you still need more information. Kindly visit the link about [Typechecking With PropTypes](https://reactjs.org/docs/typechecking-with-proptypes.html)